onnxscript/function_libs/torch_lib/graph_building_test.py

```diff
@@ -118,6 +118,12 @@ def outer(x: FLOAT[1, 2, 3]):
         expected = outer.to_model_proto()
         onnxscript.testing.assert_isomorphic(traced, expected)
 
+    def test_add_input_works_with_optionaltype(self):
```

Comment:
Would be nice to clarify what "works with" mean? Does it mean None can now be an input to the graph, or something else?

Reply:
How about test_add_input_with_optionaltype_does_not_raise_torch_internal_error

Reply:
SG!